src/kili/services/plugins/model.py

```diff
@@ -35,7 +36,7 @@ def __init__(
         if logger:
             self.logger = logger
         else:
-            self.logger = logging.getLogger()
```

Comment:
The default logger is not outputting debug/info messages while developing the plugin